apps/feedahelp/src/components/ui/features/hero/Hero.tsx

```diff
@@ -191,9 +191,9 @@ const Hero = () => {
                     src={urlForThumbnail(item.image)}
                     alt="hero-image"
                   />
-                  <div className="absolute top-1 z-12">
+                  <div className="absolute top-0 z-12">
                     <Styled.LogoImage
-                      src={"/static/images/feedahelp/logo.png"}
+                      src={"/static/images/feedahelp/FeedAhelp.png"}
                       alt="logo"
                       width={200}
```

Comment:
Here is your issue?
width: 150
height: 50

try it out

Reply:
I tried that but It not working in my case.